packages/frontend/src/models/Network.ts

```diff
@@ -22,9 +24,11 @@ class Network {
   readonly imageUrl: string
   readonly provider: ethers.providers.Provider
   readonly rpcUrl: string
-  readonly networkId: string
+  readonly networkId: number
+  readonly chainId: number
```

Comment:
what's the difference between `networkId` and `chainId` here?

Reply:
this is intermediary, just to avoid any issues where networkId may be used. eventually we should consolidate `Network` and `Chain` models.